src/api/app/views/webui/shared/_sign_up.html.haml

```diff
@@ -24,5 +24,6 @@
         class: 'form-control', required: true)
     = hidden_field_tag 'register', 'true'
     = submit_tag submit_btn_text, class: 'btn btn-primary'
-    or
-    = link_to('Log In', new_session_path)
+    - if User.session.nil?
```

Comment:
We tend to always check only for `User.session`, so without the `.nil?`. `User.session` is either a `User` or `nil`. This line could be written like this:

```suggestion
    - unless User.session
```